package.json

```diff
@@ -19,19 +19,42 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "eslint .",
-    "release": "standard-version"
+    "release": "standard-version",
+    "test": "jest"
```

Comment:
I ran the test suite and everything passed, however it is throwing this warning at the end:

![image](https://user-images.githubusercontent.com/8988697/87095658-20465680-c20f-11ea-9a8c-63ae1f52c86d.png)

Reply:
Not sure about this -- the tests are somewhat cribbed from `html-loader`'s, so maybe @evilebottnawi has more insight on what's happening. (Actually, it might be ideal if there were a shared loader testing library for `webpack-contrib` repos....?)